mobile_verifier/src/rewarder.rs

```diff
@@ -214,15 +213,11 @@ impl Rewarder {
 
         let mut transaction = self.pool.begin().await?;
 
-        // Clear the heartbeats table of old heartbeats:
-        sqlx::query("DELETE FROM heartbeats WHERE truncated_timestamp < $1")
-            .bind(reward_period.start)
-            .execute(&mut transaction)
-            .await?;
-
-        // clear the db of data sessions data & subscriber location data for the epoch
-        data_session::clear_hotspot_data_sessions(&mut transaction, reward_period).await?;
-        // subscriber_location::clear_location_shares(&mut transaction, reward_period).await?;
+        // clear out the various db tables
+        heartbeats::clear_heartbeats(&mut transaction, &reward_period.start).await?;
```

Comment:
Why do we clear all the other records from the end of the reward period and before, but the heartbeats we clear from the _start_ of the period and prior? That would leave the heartbeats from the each reward period in the DB for the next reward period’s evaluation each day, no?

Reply:
Both heartbeats and speedtests were meant to use the epoch start, not end.  Speedtests clearing now fixed:
addressed here: 67effc3558d2cd2c56ef07956dba0b19b02ffd95